repos/system_upgrade/el7toel8/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -1,20 +1,23 @@
 import itertools
 import os
 
+import yum
 from leapp.exceptions import StopActorExecutionError
 from leapp.libraries.actor import constants
 from leapp.libraries.common import dnfplugin, mounting, overlaygen, rhsm, utils
 from leapp.libraries.common.config import get_product_type
-from leapp.libraries.stdlib import CalledProcessError, api, run, config
-from leapp.models import (RequiredTargetUserspacePackages, SourceRHSMInfo, StorageInfo, TargetRepositories,
-                          TargetUserSpaceInfo, UsedTargetRepositories, UsedTargetRepository, XFSPresence)
+from leapp.libraries.stdlib import CalledProcessError, api, config, run
+from leapp.models import (RequiredTargetUserspacePackages, SourceRHSMInfo,
+                          StorageInfo, TargetRepositories, TargetRHSMInfo,
```

Comment:
Obviously you plan to remove TargetRHSMInfo as there is currently not any consumer. We have not yet the deprecation process, so feel free to remove it if we do not have use for that.

Reply:
Yeah, I've removed it.